test/Core.Test/AutoFixture/CollectionAccessSelectionFixtures.cs

```diff
@@ -0,0 +1,37 @@
+﻿using System.Reflection;
+using AutoFixture;
+using AutoFixture.Xunit2;
+using Bit.Core.Models.Data;
+
+namespace Bit.Core.Test.AutoFixture;
+
+public class CollectionAccessSelectionCustomization : ICustomization
+{
+    public bool Manage { get; set; }
+
+    public CollectionAccessSelectionCustomization(bool manage)
+    {
+        Manage = manage;
+    }
+
+    public void Customize(IFixture fixture)
+    {
+        fixture.Customize<CollectionAccessSelection>(composer => composer
+            .With(o => o.Manage, Manage));
+    }
+}
+
+public class CollectionAccessSelectionAttribute : CustomizeAttribute
```

Comment:
nit: Most of our customization attributes end with `Customize`, can we update this to follow that same pattern?